src/screens/login/HSwitchLoginFlow/HyperSwitchAuth.res

```diff
@@ -252,7 +252,7 @@ let make = (~setAuthStatus: HyperSwitchAuthTypes.authStatus => unit, ~authType,
             | _ => React.null
             }}
           </div>
-          {note(authType, setAuthType, isMagicLinkEnabled)}
+          <div id="card-foot-text"> {note(authType, setAuthType, isMagicLinkEnabled)} </div>
```

Comment:
Use AddDataAttributes for adding adding any attribute like id to the next div rather than adding id directly to the div
```
 <AddDataAttributes attributes=[("id","card-foot-text)]>
</AddDataAttributes>
```